instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -50,6 +50,31 @@ def hello():
     application = get_wsgi_application()
     application = OpenTelemetryMiddleware(application)
 
+Usage (Web.py)
+--------------
+
+.. code-block:: python
+
+    import web
+    from opentelemetry.instrumentation.wsgi import OpenTelemetryMiddleware
+    from cheroot import wsgi
+
+    urls = (
+        '/', 'index'
+    )
+    class index:
+        def GET(self):
+            return "Hello, world!"
+
+    if __name__ == "__main__":
+	    app = web.application(urls, globals())
+	    func = app.wsgifunc()
+
+	    func = OpenTelemetryMiddleware(func)
+
+	    server = wsgi.WSGIServer(("localhost", 5100),func, server_name="localhost")
+	    server.start()
```

Comment:
```suggestion
    import web
    from opentelemetry.instrumentation.wsgi import OpenTelemetryMiddleware
    from cheroot import wsgi

    urls = ('/', 'index')


    class index:

        def GET(self):
            return "Hello, world!"


    if __name__ == "__main__":
        app = web.application(urls, globals())
        func = app.wsgifunc()

        func = OpenTelemetryMiddleware(func)

        server = wsgi.WSGIServer(
            ("localhost", 5100), func, server_name="localhost"
        )
        server.start()
```

This code mixes tabs and spaces among other things, please use this suggestion instead.